docs/architecture/data-pipelines.md

```diff
@@ -0,0 +1,104 @@
+---
+parent: Architecture
+---
+
+# Data Pipelines
+{: .no_toc }
+
+## Table of Contents
+{: .no_toc .text-delta }
+
+- Table of Contents
+{:toc}
+
+
+### Summary
+
+This document outlines how data flows function in OCW-next prior to the rework of the architecture. It serves as a baseline to inform our decision making and scoping of those efforts. Currently some of the pipeline is manual, and some of it is not the most time-efficient.
+
+### Desired Architecture Qualities
+
+- **Decentralized**
+  - Empower course authors to create and publish content in a non-centralized way
+- **Versioned**
+  - Support course authors to be able to make changes without losing historical versions
+  - Support course authors being able to revert to or work off of older versions of course content
+  - Support course authors being able to work on a non-published draft version of a course
+- **Previewable**
+  - Support course authors being able to preview their course material as it will appear when published
+- **Simplicity**
+  - We're aiming to build a favorable user experience for course authors that gets out of their way and makes their jobs as easy as possible
+  - The system should favor simplifying or limiting complex functionality, but design and implement with future flexibility in mind
+
+
+### Current Architecture
+
+| ![Current Architecture Diagram](assets/OCW-next-current.png) |
+|:--:|
+| [full version](assets/OCW-next-current.png) |
+
+Major components here are:
+
+- **OCW V1** - running on Plone
+- **MIT Open:**
+  - Manages normalization of Plone data
+  - Stores normalized data in-memory in Postgres
+  - Indexes normalized data in-memory to Elasticsearch
+- **OCW-next:**
+  - In-progress implementation of the next version of OCW, using [Hugo](https://gohugo.io/)
+  - Deployment of updated course content is manual
+
+### Final Architecture
+
+| ![Final Architecture Diagram](assets/OCW-next-final.png) |
+|:--:|
+| [full version](assets/OCW-next-final.png) |
+
+- **ocw-sync**
+  - Lightweight Python script running `ocw-data-parser`
+  - Enables rapid iteration of `ocw-data-parser` not bound to Open releases
+  - The life expectancy of this is only until OCW-v1 is decommissioned
+- **ocw-to-hugo**
+  - This will still be run manually, since we only need to run it infrequently
```

Comment:
Maybe add a short description here about what `ocw-to-hugo` does, something like:

 - Responsible for consuming normalized OCW JSON and producing Markdown for use with `hugo-course-publisher`